src-docs/src/views/header/header_dark.tsx

```diff
@@ -56,7 +56,7 @@ export default ({ theme }: { theme: any }) => {
           {
             items: [
               <OuiBadge
-                color={theme.ouiColorDarkestShade.rgba}
+                color={theme.ouiColorDarkestShade}
```

Comment:
what changed here?

Reply:
My script doesn't break colors into `r`, `g`, `b`, `rgb`, and `rgba`. The previous implementation did.

Reply:
That sounds like it may be a breaking change if the shades are used with those color breakdowns in OSD. But I think it's small enough for us to fix on that side if need be.

Reply:
quick code search looks like we're OK.